sdk/log/simple.go

```diff
@@ -80,3 +136,8 @@ func (s *SimpleProcessor) ForceFlush(ctx context.Context) error {
 type SimpleProcessorOption interface {
 	apply()
 }
+
+// ResetSimpleProcessorIDCounterForTesting resets the global ID counter for testing purposes.
+func ResetSimpleProcessorIDCounterForTesting() {
```

Comment:
I don't really recommend exposing a method externally just for unit testing purposes. Once it's used externally, we'll need to maintain that method.